mifospay/src/main/java/org/mifos/mobilewallet/mifospay/savedcards/ui/CardsFragment.java

```diff
@@ -103,6 +107,7 @@ private void setUpSwipeRefresh() {
             public void onRefresh() {
                 getSwipeRefreshLayout().setRefreshing(false);
                 mCardsPresenter.fetchSavedCards();
+                addCard.setVisibility(View.VISIBLE);
```

Comment:
Instead of making the `addCard` visible here, it would be much better to make it visible only when the `mCardsPresenter.fetchSavedCards()` the response is successful i.e. inside `showSavedCards` function

Reply:
Okay